enterprise/migrations/0104_auto_20200723_1418.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.2.14 on 2020-07-23 19:18
```

Comment:
You should only need one migration for adding this field as we want it to default to `False` right? Also a nit that I usually like naming migration files to give a sense of what they do